models/stg_reddit_ads__account_report.sql

```diff
@@ -1,9 +1,9 @@
+{{ config(enabled=var('ad_reporting__reddit_ads_enabled', True)) }}
```

Comment:
This config will need to be added to every model within this (and the transform) package. This ensures that users leveraging the Ad Reporting package will be able to turn this package off if they are not using it.

Reply:
Please add to the other models.

Reply:
done, have it added for transforms, forgot to do it here